docs/src/content/docs/fr/resources/community-content.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Contenu communautaire
+description: Découvrez les guides, articles et vidéos produits par la communauté pour vous aider à apprendre et à créer avec Starlight !
+---
+
+:::tip[Ajoutez le vôtre !]
+Avez-vous produit du contenu sur Starlight ?
+Ouvrez une PR ajoutant un lien à cette page !
+:::
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+## Articles et revues
+
+Voici une collection de publications et d'articles pour en apprendre plus sur Starlight et les expériences d'autres personnes :
+
+<CardGrid>
+	<LinkCard
+		href="https://devm.io/open-source/starlight-astro"
+		title="Génération de sites statiques avec Starlight"
+		description="« Aucune idée n'est trop grande ou trop petite lors de la conception de composants » — une entrevue avec Chris Swithinbank, responsable de Starlight"
+	/>
+	<LinkCard
+		href="https://frontendatscale.com/blog/hybrid-frontend-architecture/"
+		title="Une architecture frontend hybride avec Astro et Starlight"
```

Comment:
Decided to not translate "frontend" as I don't know how, never heard a translation for it and also https://fr.wiktionary.org/wiki/frontend